external-crates/move/crates/move-compiler/src/parser/syntax.rs

```diff
@@ -336,6 +340,29 @@ where
     }
 }
 
+// Helper for location blocks
+
+macro_rules! ok_with_loc {
+    ($context:expr, $body:block) => {{
```

Comment:
Are the blocks not expressions?

Reply:
I have had rust complain before that they are not, but it does appear to matter situation-dependent and works just fine here